src/Parsers/Streaming/ASTSessionRangeComparision.cpp

```diff
@@ -21,26 +30,24 @@ void ASTSessionRangeComparision::updateTreeHashImpl(SipHash & hash_state) const
 
 void ASTSessionRangeComparision::formatImplWithoutAlias(const FormatSettings & settings, FormatState &, FormatStateStacked) const
 {
-    if (children.size() == 2)
-    {
-        settings.ostr << (settings.hilite ? hilite_keyword : "") << (start_with_inclusion ? "[" : "(") << (settings.hilite ? hilite_none : "");
-        children[0]->format(settings);
-        settings.ostr << (settings.hilite ? hilite_keyword : "") << "," << (settings.hilite ? hilite_none : "");
-        children[1]->format(settings);
-        settings.ostr << (settings.hilite ? hilite_keyword : "") << (end_with_inclusion ? "]" : ")") << (settings.hilite ? hilite_none : "");
-    }
+    assert(children.size() == 2);
+
+    settings.ostr << (settings.hilite ? hilite_keyword : "") << (start_with_inclusion ? "[" : "(") << (settings.hilite ? hilite_none : "");
+    children[0]->format(settings);
+    settings.ostr << (settings.hilite ? hilite_keyword : "") << "," << (settings.hilite ? hilite_none : "");
+    children[1]->format(settings);
+    settings.ostr << (settings.hilite ? hilite_keyword : "") << (end_with_inclusion ? "]" : ")") << (settings.hilite ? hilite_none : "");
 }
 
 void ASTSessionRangeComparision::appendColumnNameImpl(WriteBuffer & ostr) const
 {
-    if (children.size() == 2)
-    {
-        writeString((start_with_inclusion ? "[" : "("), ostr);
-        children[0]->appendColumnName(ostr);
-        writeString(", ", ostr);
-        children[1]->appendColumnName(ostr);
-        writeString((end_with_inclusion ? "]" : ")"), ostr);
-    }
+    assert(children.size() == 2);
+
+    writeString((start_with_inclusion ? "[" : "("), ostr);
+    children[0]->appendColumnName(ostr);
+    writeString(", ", ostr);
+    children[1]->appendColumnName(ostr);
+    writeString((end_with_inclusion ? "]" : ")"), ostr);
```

Comment:
why we remove this condition check? is there any reason here?

Reply:
it has only two elements, guaranteed in `ParserSessionRangeComparisonExpressionIfPossible::parseImpl`